MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Indirect/ISISCalibration.ui

```diff
@@ -92,7 +92,17 @@
           </property>
          </widget>
         </item>
-        <item>
+         <item>
+           <widget class="QCheckBox" name="ckLoadLogFiles">
+             <property name="text">
+               <string>Load Log Files</string>
+             </property>
+             <property name="checked">
+               <bool>true</bool>
```

Comment:
This should be `false` by default. I think once this change is made we are good to go!